client/components/autocomplete_list/test/index.coffee

```diff
@@ -21,6 +21,8 @@ describe 'AutocompleteList', ->
         )
       $.fn.typeahead = sinon.stub()
       @AutocompleteList = benv.require resolve __dirname, '../index'
+      DragContainer = benv.require resolve __dirname, '../../drag_drop/index'
+      @AutocompleteList.__set__ 'DragContainer', React.createFactory DragContainer
```

Comment:
Curious, is it necessary to createFactory manually in this test? I would think that initializing AutocompleteList will create the component automatically. Maybe if we're doing something special to DragContainer but it seems like we're not here?